rpc-rs/codegen.toml

```diff
@@ -2,7 +2,9 @@
 
 [[models]]
 # wasmbus-core
-url = "https://cdn.jsdelivr.net/gh/wasmcloud/interfaces@2c89bb706c6f2785a926dcde78ebc6a511a33206/core/wasmcloud-core.smithy"
+# url = "https://cdn.jsdelivr.net/gh/wasmcloud/interfaces@2c89bb706c6f2785a926dcde78ebc6a511a33206/core/wasmcloud-core.smithy"
```

Comment:
Will we have to update this to the commit hash?

Reply:
I'm not sure actually. I `wasmbus-core` was hash-based, but immediately below, `wasmbus-model` was based off main. I can't find `2c89bb70` in the repo at all, so I'm not sure what this is pointing to currently

Reply:
Updated to point to main